src/actions.ts

```diff
@@ -27,38 +27,98 @@ When('I type {string} to {string}', async function (value: string, alias: string
     await element.addValue(typeValue);
 });
 
+/**
+ * Click implementation
+ * @param alias - alias of element to click
+ * @param enableWait - enable clickability wait
+ */
+async function click(alias: string, enableWait: boolean): Promise<void> {
```

Comment:
from my point of view such functions should be moved to the separate module with actions and also I would suggest to add true default value for `enableWait` parameter

Reply:
good idea, I will eject it into separate module.